src/lib/svg_text_utils.js

```diff
@@ -196,47 +203,83 @@ function texToSVG(_texString, _config, _callback) {
             },
             displayAlign: 'left',
         });
-    },
-    function() {
+    };
+
+    var v2SetRenderer = function() {
         // Get original renderer
         originalRenderer = MathJax.Hub.config.menuSettings.renderer;
         if(originalRenderer !== 'SVG') {
             return MathJax.Hub.setRenderer('SVG');
         }
-    },
-    function() {
+    };
+
+    var initiateMathJax = function() {
         var randomID = 'math-output-' + Lib.randstr({}, 64);
         tmpDiv = d3.select('body').append('div')
             .attr({id: randomID})
             .style({visibility: 'hidden', position: 'absolute'})
             .style({'font-size': _config.fontSize + 'px'})
             .text(cleanEscapesForTex(_texString));
 
-        return MathJax.Hub.Typeset(tmpDiv.node());
-    },
-    function() {
+        var tmpNode = tmpDiv.node();
+
+        if(MathJaxVersion < 3) {
+            return MathJax.Hub.Typeset(tmpNode);
+        } {
+            return MathJax.typeset([tmpNode]);
+        }
```

Comment:
Interesting, I didn't know you could make a block like this second one, ie with no control statement preceding it. I suppose there's a nice symmetry to this, as the two typeset statements are equally indented, but it's confusing to read so I'd rather we remove it.
```suggestion
        if(MathJaxVersion < 3) {
            return MathJax.Hub.Typeset(tmpNode);
        }
        return MathJax.typeset([tmpNode]);
```